### PR TITLE
Url change on article

### PR DESCRIPTION
**Pull request by Jcoetsee**
This is in reference to https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2328

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbPartnerCertification/certification/plan-wifi.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Microsoft’s approach to drive quality and agility into the wireless networks c
 [Ensuring an Enterprise Class Wireless Skype for Business Experience](https://www.microsoft.com/en-us/download/details.aspx?id=47257)
```
Comment (nenonix) [inline Skype/SfbPartnerCertification/certification/plan-wifi.md]:
```suggestion
[Ensuring an Enterprise Class Wireless Skype for Business Experience](https://www.microsoft.com/download/details.aspx?id=47257)
```

**Comment by Jcoetsee** [comment]
@nenonix I have looked all over, the end user wants that link back in the original MS article since they want to refer their clients to it. I don't see any other location for this unless the article can be saved and rather uploaded directly onto the MS side of things?

**Comment by scanum** [comment]
@jambirk Hi James, changes look correct. Could you please review and merge? Thanks!

**Comment by jambirk** [comment]
Closing the PR, we are removing the section rather than updating the link.

**Pull request closed without merging**